internal/host/transition.go

```diff
@@ -74,6 +74,7 @@ func (th *transitionHandler) PostRegisterHost(sw stateswitch.StateSwitch, args s
 
 	sHost.host.StatusUpdatedAt = strfmt.DateTime(time.Now())
 	sHost.host.StatusInfo = swag.String(statusInfoDiscovering)
+	sHost.host.InstallerArgs = "--delete-karg 'console=tty0'"
```

Comment:
should be ttyS0

Reply:
or perhaps delete it via regexp

Reply:
According to the decision from the [remove console=ttyS0 on metal issue](https://github.com/coreos/fedora-coreos-tracker/issues/567#issuecomment-662632315) - all console defaults kargs can be dropped.
But as looking in the [coreos-installer code](https://github.com/jlebon/coreos-installer/blob/fd6be04d65e1a45bcd46513fbfbbcc44122f30dd/src/install.rs#L273-L278) it seems regex isn't implemented.

Reply:
Anyhow, I created a [PR for coreos-installer repository](https://github.com/coreos/coreos-installer/pull/414).
I guess it would take time.